en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: news_post
+title: "Multiple vulnerabilities in RubyGems"
+author: "usa"
+translator:
+date: 2018-02-17 03:00:00 +0000
+tags: security
+lang: en
+---
+
+There are multiple vulnerabilities in RubyGems bundled by Ruby.
+It is [reported at the official blog of RubyGems](http://blog.rubygems.org/2018/02/15/2.7.6-released.html).
+
+## Details
+
+The following vulnerabilities have been reported.
+
+* Prevent path traversal when writing to a symlinked basedir outside of the root.
+* Fix possible Unsafe Object Deserialization Vulnerability in gem owner.
+* Strictly interpret octal fields in tar headers.
+* Raise a security error when there are duplicate files in a package.
+* Enforce URL validation on spec homepage attribute.
+* Mitigate XSS vulnerability in homepage attribute when displayed via gem server.
+* Prevent Path Traversal issue during gem installation.
```

Comment:
Do any of above has CVE number? Maybe we should reference their CVE number.

Reply:
They (RubyGems team) does not get CVE numbers now.